README.md

```diff
@@ -982,7 +982,7 @@
     // declaration to the top of the scope,
     // which means our example could be rewritten as:
     function example() {
-      let declaredButNotAssigned;
+      var declaredButNotAssigned;
```

Comment:
Using `let` here, which is not hoisted, is the intention, since it's meant to represent what happens when `var` is hoisted on line 978.